README.md

```diff
@@ -33,9 +24,10 @@
 <!-- - [❓ FAQ](#faq) -->
 - [📝 License](#license)
 
-# 📰 My Portfolio <a name="about-project"></a>
+# 📰 Capstone Project <a name="about-project"></a>
 
-- My Portfolio project is a project aimed at building my professional portfolio and also showcase my works and skills as a web developer.
+- This project is a website for a Javascript conference that hold bi-annually. It showcases all the speakers for the conference, date, venue, sponsors and the program schedule for the conference
+  [link to Short Description](https://www.loom.com/share/37a9fcd1a69544bcbbbd7ae961a6fcb0)
```

Comment:
- [ ] As recommended by the previous reviewer, please show us some piece of code that you liked the most not necessarily the feature but some lines of code from your editor or even on your GitHub repo that you are proud of :sunglasses: :cool: